checkInput.ts

```diff
@@ -0,0 +1,35 @@
+let isValid: boolean[] = []
+
+for (let messageIndex = 0; messageIndex < numberOfMessages; messageIndex++) {
+    isValid.push(false)
+}
+
+function checkRowInput (messageIndex: number): void {
```

Comment:
It's not conventional to have whitespace between the function name and it's arg list, but it is between the arg list and the return type. I think more conventionally this would be:

`function checkRowInput(messageIndex: number) : void {`